### PR TITLE
Excluding the Nuglify package from Appveyor and upgrading Nuglify

### PR DESCRIPTION
**Pull request by RapidOwl**

### CODE REVIEW

**Comment by RapidOwl** [comment]
@Danny-UKDM This is a good point. I just updated the value in the dependencies file. No idea if it'll still work. I hate Paket so very much.

File: appveyor.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ deploy:
     api_key:
       secure: 4yuwHXUFFa5ijOkkvvmEX/SjKhzWTvln7JtYrgVB1KdXcpF4o8WFAeNm6PTlwN+L
     skip_symbols: true # Will be uploaded by former deployment job
+    artifact: /^((?!MustardBlack\.Assets\.Nuglify)(.*))\.nupkg/
```
Comment (ozwaldDunlop) [inline appveyor.yml:38]: this only seems to match "core" MB nupkg
image - <https://regex101.com/r/i3JgNL/1>
I assumed you were trying to get it to upload all apart from nuglify

Comment (ozwaldDunlop) [reply appveyor.yml:38]: my mistake. example link didn't have multiline on

Comment (Danny-UKDM) [reply appveyor.yml:38]: This works as expected when these flags are enabled
image
Can we include those flags on the `artifact` regex?

**Comment by RapidOwl** [comment]
@Danny-UKDM Ollie offered a shorter solution so I've used that instead for future readability.

**Pull request merged**